django/core/checks/registry.py

```diff
@@ -16,6 +16,7 @@ class Tags(object):
     security = 'security'
     signals = 'signals'
     templates = 'templates'
+    sessions = 'sessions'
```

Comment:
I don't like putting values for contrib apps in here. Maybe it could be something like an app registry property instead?